src/assets/lima-config.yaml

```diff
@@ -48,13 +48,15 @@ provision:
   mode: system
   script: |
     #!/bin/sh
-    set -o errexit -o nounset -o xtrace
-    ETH0="$(ip route | grep "^default.* dev eth0 ")"
-    RD0="$(ip route | grep "^default.* dev rd0 ")"
-    IP="$(echo "$RD0" | awk '{print $3}')"
-    METRIC="$(echo "$ETH0" | awk '{print $NF}')"
-    ip route del default via "$IP" dev rd0
-    ip route add default via "$IP" dev rd0 metric $((METRIC - 1))
+    if [ "$(uname)" == 'Darwin' ]; then
+      set -o errexit -o nounset -o xtrace
+      ETH0="$(ip route | grep "^default.* dev eth0 ")"
+      RD0="$(ip route | grep "^default.* dev rd0 ")"
```

Comment:
Sorry, I know I suggested the `uname` test, but it is not going to work: the provisioning script runs inside the VM, so will always be Linux.

Instead we can check if the `rd0` interface exists, and only try to manipulate it's metric when it does:

```shell
if [ -n "${RD0}" ]; then
```

Reply:
Done